build.gradle

```diff
@@ -1,194 +1,203 @@
 plugins {
-	id "java"
 	id "java-library"
 	id "checkstyle"
 	id "maven-publish"
 	id "me.modmuss50.remotesign" version "0.1.0"
 	id "org.cadixdev.licenser" version "0.6.1"
 }
 
-group "net.fabricmc"
-archivesBaseName = "access-widener"
-
-def ENV = System.getenv()
-version = "2.1.0"
-logger.lifecycle("Building AccessWidener: " + version)
-
-repositories {
-	mavenCentral()
-
-	maven {
-		name = "Fabric"
-		url = "https://maven.fabricmc.net/"
+allprojects {
```

Comment:
The maven (central) publishing is going to be wrong just doing this, let me know if you want me to handle this.

Reply:
I have no idea how it works ;p

Reply:
Leave this to me 👍 might be a few days as I’m quite busy IRL.